src/cmd_all/src/single_node.rs

```diff
@@ -227,3 +268,254 @@ impl SingleNodeOpts {
+                            etcd_password: [REDACTED alloc::string::String],
+                            sql_endpoint: Some(
+                                "sqlite:///Users/noelkwan/.risingwave/meta_store/single_node.db?mode=rwc",
+                            ),
+                            dashboard_ui_path: None,
+                            prometheus_endpoint: None,
+                            prometheus_selector: None,
+                            connector_rpc_endpoint: None,
+                            privatelink_endpoint_default_tags: None,
+                            config_path: "",
+                            backend: Some(
+                                Sql,
+                            ),
+                            barrier_interval_ms: None,
+                            sstable_size_mb: None,
+                            block_size_kb: None,
+                            bloom_false_positive: None,
+                            state_store: Some(
+                                "hummock+fs:///root/.risingwave/state_store",
+                            ),
+                            data_directory: Some(
+                                "some path with spaces",
+                            ),
+                            do_not_config_object_storage_lifecycle: None,
+                            backup_storage_url: None,
+                            backup_storage_directory: None,
+                            heap_profiling_dir: None,
+                        },
+                    ),
+                    compute_opts: Some(
+                        ComputeNodeOpts {
+                            listen_addr: "127.0.0.1:8888",
+                            advertise_addr: Some(
+                                "0.0.0.0:5688",
+                            ),
+                            prometheus_listener_addr: "127.0.0.1:1222",
+                            meta_address: List(
+                                [
+                                    http://127.0.0.1:5690/,
+                                ],
+                            ),
+                            connector_rpc_endpoint: None,
+                            connector_rpc_sink_payload_format: None,
+                            config_path: "",
+                            total_memory_bytes: 123,
+                            parallelism: 10,
+                            role: Both,
+                            metrics_level: None,
+                            data_file_cache_dir: None,
+                            meta_file_cache_dir: None,
+                            async_stack_trace: Some(
+                                ReleaseVerbose,
+                            ),
+                            heap_profiling_dir: None,
+                        },
+                    ),
+                    frontend_opts: Some(
+                        FrontendOpts {
+                            listen_addr: "0.0.0.0:4566",
```

Comment:
Default value we specified in `src/frontend/src/lib.rs` is `127.0.0.1:4566`.

But in standalone mode, we specify `0.0.0.0:4566`. So the node level defaults do not override as well.